Description: Set correct path to file with color names in manpage
Author: Decklin Foster <decklin@red-bean.com>
Origin: vendor
Bug-Debian: http://bugs.debian.org/501728
Forwarded: no
Reviewed-By: Ryan Kavanagh <ryanakca@kubuntu.org>
Last-Update: 2011-03-06

--- a/doc/rxvt.1.pod
+++ b/doc/rxvt.1.pod
@@ -1790,7 +1790,7 @@ If set and accessible, gives the name of
 
 =over 4
 
-=item B</usr/lib/X11/rgb.txt>
+=item B</etc/X11/rgb.txt>
 
 Colour names.
 
